packages/cosmwasm/src/signingcosmwasmclient.ts

```diff
@@ -200,10 +204,12 @@ export class SigningCosmWasmClient extends CosmWasmClient {
 
   private readonly signer: OfflineSigner;
   private readonly aminoTypes: AminoTypes;
-  private readonly gasPrice: GasPrice | undefined;
+  private readonly gasPrice: GasPrice | DynamicGasPriceConfig | undefined;
   // Starting with Cosmos SDK 0.47, we see many cases in which 1.3 is not enough anymore
   // E.g. https://github.com/cosmos/cosmos-sdk/issues/16020
   private readonly defaultGasMultiplier = 1.4;
+  // Default multiplier for dynamic gas price (applied on top of queried price)
+  private readonly defaultDynamicGasMultiplier = 1.3;
```

Comment:
This is a gas **price** multiplier, not a gas multiplier, right? I think it is important to highlight this e.g. by renaming

```suggestion
  private readonly defaultDynamicGasPriceMultiplier = 1.3;
```